webpack/assets/javascripts/react_app/common/MountingService.js

```diff
@@ -17,12 +24,21 @@ export function mount(component, selector, data, flattenData = false) {
   }
 }
 
-export function mountNode(component, reactNode, data, flattenData = false) {
+export function mountNode(
+  component,
+  reactNode,
+  data,
+  metadata,
+  flattenData,
```

Comment:
This no longer defaults to false here

Reply:
`mountNode` is an internal function, we should drop the `export` from it though. so no need for default value if it is already given by `mount`